Mapbox/MapboxMapsFoundation/BaseMapView.swift

```diff
@@ -63,7 +63,7 @@ open class BaseMapView: UIView, MapClient, MBMMetalViewProvider {
 
     /// The map's current center coordinate.
     public var centerCoordinate: CLLocationCoordinate2D {
-        return cameraView.centerCoordinate
+        return cameraView.centerCoordinate.wrap()
```

Comment:
`cameraView.centerCoordinate.longitude` must be allowed to be > 180 and < -180 so that the Core Animation-based interpolation works correctly.

Reply:
maybe I'm interpreting something incorrectly but when going through the chain, it looks like the wrap() function normalizes the coordinate to be with in -180..180.

Reply:
> when going through the chain

Which chain?

Reply:
Like I stepped into the wrap function which eventually took me [here](https://github.com/mapbox/mapbox-maps-ios/blob/main/Mapbox/MapboxMapsFoundation/Extensions/Core/Utils.swift#L36)

Reply:
Added a comment to capture what we discussed.